### DISCUSSION TITLE
Not able to setup App Layout (SubNav Tabs to be specific) in non-angular app

### DISCUSSION THREAD

**Discussion by rajanbhirud** [vmware-clarity/core#83] [Q&A] [answered]
I am using @cds/core and @cds/react in my react app to create a UI. I am unable to find any blogs/documentations on how I can set up App Layout (Header, SideNav, SubNav) with @cds/core and @cds/react. Is it not supported?

I have tried this https://clarity.design/foundation/app-layout/, it didn't work.

**Comment by ashleyryan** [comment, marked as answer]
Hello,

Clarity Core doesn't yet have a header or subnav component. The App Layout page you linked to is in reference to the older html/css and Angular implementations of the design system. 

You can use `@clr/ui` to get the header and subnav (and use it in conjunction with the Core CSS shim - which is intended to update the colors of clr/ui and clr/angular to match the updated Core colors), but `@clr/ui` has some global CSS that you may need to unset to get the vertical nav to display properly.

I put together a demo here:

https://stackblitz.com/edit/clarity-react-bdp6ye?file=src%2FApp.tsx,src%2Findex.css,src%2Findex.html
